tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/SourcelessTestExecutionTests.java

```diff
@@ -25,8 +25,8 @@ public void setup() throws BallerinaTestException {
     // Scenario : Modules only have tests and no sourcefiles. Default module doesnt have sources or tests
     @Test()
     public void test_SourcelessModule_TestExecution() throws BallerinaTestException {
-        String msg1 = "[pass] test1";
-        String msg2 = "[pass] test2";
+        String msg1 = "sourcelessModulesTest.module1\n\n\n\t\t1 passing";
```

Comment:
Why do we show the passing list here?

Reply:
I added this in because in case something fails, we couldn't know. earlier we asserted the `[pass]` message but since we are removing that, we wouldn't know if the test has actually failed or not

Reply:
If a data-driven test is run, again the UX issue will be seen. @hevayo WDYT?